### DISCUSSION TITLE
Mouse Pointer Support for mobile

### DISCUSSION THREAD

**Discussion by nckrwlmn** [Ylianst/MeshCentral#6129] [Ideas]
Teamviewer mobile app has a mouse mode where the pointer is dragged around the screen by dragging your finger on the screen. It emulates a touchpad with the phone screen. Is it possible to do that on the browser? If this had a mobile friendly mouse mode like that it could replace teamviewer any day.

**Comment by nckrwlmn** [comment]
Hello? Has anyone here used the remote mouse pointer feature? Its kind of a must have for remote controlling a PC from a phone.

**Reply by si458** [reply]
This feature doesn't exist, it needs adding first
